core/src/xyz/magicjourney/nebulaquest/dice/Dice.java

```diff
@@ -28,7 +31,14 @@ public Dice(AssetManager assets) {
     this.setWidth(400);
     this.setHeight(200);
     this.setVisible(false);
+    this.rollNumber = 0;
 
+    this.debugRolls = new int[] {
+      5, 5, 5,
+      5, 2, 
+      5, 6,
+      5, 12
+    };
```

Comment:
Nie do końca rozumiem, za co odpowiada ta tablica.

Reply:
Ustawia ona rzuty kostką. Używamy jej w metodzie debugRoll. W tym ustawieniu pierwsze trzy rzuty kostką zwrócą 5. A kolejne 5, 2, 5, 6, 5, 12. To konkretne ustawienie testuje unknown jump. Pierwsze trzy piątki są nam potrzebne, aby przenieść graczy na pole teleportu. Kolejna piątka wchodzi na pole unkown jump. Więc po kolei gracz pierwszy wyrzuci 5, a następnie 2, co przeniesie go na teleport zachodni, Kolejny 5 i sześć co przeniesie go na start, a ostatni 5 i 12 co wywali go niestety do nebuli. 

Widać to ładnie na tym nagraniu, zwróć uwagę na to, że wyniki kostki to kolejne elementy tej tablicy.

https://github.com/Our-Magic-Journey/NebulaQuest/assets/13831258/4db581a4-b609-459a-b96b-36c382c83c4d